internal/resources/selfmonitor/resources.go

```diff
@@ -364,6 +372,10 @@ func makeService(name types.NamespacedName, port int) *corev1.Service {
 			Name:      name.Name,
 			Namespace: name.Namespace,
 			Labels:    defaultLabels(name.Name),
+			Annotations: map[string]string{
+				"prometheus.io/scrape": "true",
+				"prometheus.io/port":   "9090",
+			},
```

Comment:
Do we only need it for benchmarking? If yes, let's not do it and deploy and additional metrics service in the script.